tasks/main.yml

```diff
@@ -132,3 +132,4 @@
   with_items:
     - absent
     - present
+  when: start_containers | default(true)
```

Comment:
If you do this, the containers will not even be created. `up` always starts. You will have to use `docker-compose create` get what you want. I'm not sure if that's even do-able with `docker_service`.